libs/redux/src/lib/stores/prompt/prompt.store.ts

```diff
@@ -156,7 +166,7 @@ export const promptStore: IPromptStore = {
         if (isCanceled()) return;
 
         ws = factory.createWebSocket(url);
-        ws.onClose(() => actions.dispose());
+        ws.onClose((evt) => actions.dispose(evt));
```

Comment:
I think you can pass a self executing function here I believe.

```suggestion
        ws.onClose(actions.dispose);
```